aten/src/ATen/Quantizer.h

```diff
@@ -16,32 +16,57 @@ namespace at {
 using QTensor = Tensor;
 using RealTensor = Tensor;
 
-// Quantizer is an object that stores all the information
-// that's necessary to perform quantize and dequantize
-// operation
+/* Quantizer is the class for storing all the information
+ * that's necessary to perform quantize and dequantize
```

Comment:
nit: I'm not sure what you are trying to get across by mentioning "quantize and dequantize operation" and "providing quantize() and dequentize() methods" -- aren't these the same things in practice?

Reply:
right.. will remove..